ra2ce/analyses/indirect/losses.py

```diff
@@ -18,17 +18,28 @@
     You should have received a copy of the GNU General Public License
     along with this program.  If not, see <http://www.gnu.org/licenses/>.
 """
-
-
+from ast import literal_eval
 from pathlib import Path
+
 import numpy as np
 import pandas as pd
+import geopandas as gpd
 
 from ra2ce.analyses.analysis_config_data.analysis_config_data import (
     AnalysisConfigData,
     AnalysisSectionIndirect,
 )
 
+trip_types = [
+    "business"
+    "commute"
+    "freight"
+    "other"
+]
+
+road_types = ["motorway", "primary", "secondary", "tertiary", "residential", "secondary_unpaved",
+              "tertiary_unpaved", "residential_unpaved", "tunnel", "bridge", "culvert"]
```

Comment:
These do not belong here! we have it defined as an enum in `ra2ce.graph.network_config_data.enums.road_type_enum.py` Please extend that enum list if required, but we do not want to use fixed strings here.